alpha/scripts/utils/enableVirtualEventPermissionToAllPartners.php

```diff
@@ -30,17 +34,23 @@
 
 $c = new Criteria();
 $c->addAscendingOrderByColumn(PartnerPeer::ID);
-$c->addAnd(PartnerPeer::ID, 99, Criteria::GREATER_EQUAL);
+$c->addAnd(PartnerPeer::ID, $firstPartner, Criteria::GREATER_EQUAL);
 $c->addAnd(PartnerPeer::STATUS,1, Criteria::EQUAL);
 $c->setLimit($countLimitEachLoop);
-
 $partners = PartnerPeer::doSelect($c, $con);
 
+$isLastPartner = false;
 while (count($partners))
 {
 	foreach ($partners as $partner)
 	{
 		/* @var $partner Partner */
+		if ($partner->getId() > $lastPartner)
```

Comment:
instead of adding all the conditions with the breaks you can add to the db query id lessThenEqual